utils/lib/README.md

```diff
@@ -1,19 +1,22 @@
 # ledgersmb-release-scripts
+
 Tools to assist LedgerSMB users and developers
 
-######Copyright (c) 20115, 2016, 2017 LedgerSMB Project
-######Written by SB Tech Services info@sbts.com.au
+<!-- markdownlint-disable MD001 -->
+<!-- markdownlint-disable MD018 -->
+<!-- markdownlint-disable MD022 -->
+###### Copyright (c) 20115, 2016, 2017 LedgerSMB Project
+###### Written by SB Tech Services info@sbts.com.au
```

Comment:
Same comment as for utils/devel/README.md
Copyright should be wrapped in a codeblock.

As the owner of SB Tech Services, I'm happy for the attribution to be unemphasised text.
Could be dropped as well I guess

Reply:
Fixed